web/package.json

```diff
@@ -68,6 +68,8 @@
     "classnames": "^2.2.6",
     "dayjs": "1.8.28",
     "dotenv": "^16.0.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-simple-import-sort": "^8.0.0",
```

Comment:
Are both of these required? I think `eslint-plugin-import` is enough

Reply:
Because I think it can auto fix import order problem.I try to use only eslint-plugin-import but no work.